src/components/ModalWindow/ModalWindow.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+import Modal from 'react-bootstrap/Modal';
+import { useAppDispatch } from '../../hooks/hooks';
+import { addModalTodo } from '../../store/reducers/TodosSlice';
+import s from "./ModalWindow.module.scss";
+
+const ModalWindow: React.FC = () => {
+
+  const [input, setInput] = useState<any>({});
+  const [show, setShow] = useState(false);
+
+  const handleClose = () => setShow(false);
+  const handleShow = () => setShow(true);
+
+  const dispatch = useAppDispatch();
+
+  const handleSubmit = (e: React.FormEvent<HTMLButtonElement>) => {
+    e.preventDefault();
+    if (/^[A-Za-zА-Яа-яЁё0-9\s]+$/.test(input.message) && /[0-9]+/.test(input.data) && /[0-9]+/.test(input.expData)) {
+      dispatch(addModalTodo(input));
+      setInput({});
+      setShow(false);
+    }
+  }
```

Comment:
check eslint or prettier configuration. Coz you don't have semicolons after methods.

Reply:
I checked and fixed it, earlier for some reason all my plugins were removed from vs code, now I installed prettier, there should be no more such problems